index.js

```diff
@@ -5,9 +5,9 @@ const oxmlContentTypes = toBytes('[Content_Types].xml');
 const oxmlRels = toBytes('_rels/.rels');
 
 module.exports = input => {
-	const buf = new Uint8Array(input);
+	const buf = input;
 
-	if (!(buf && buf.length > 1)) {
```

Comment:
It's a bit unclear why u compare with -1, why not use buf[1] !== undefined ?

Reply:
It checks that it is also a number

Reply:
Why is that needed? also `'' > -1` is `true` which is obviously not a number

Reply:
It's a check that on the way.
It's not important